org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractMultiQuickfixTest.java

```diff
@@ -65,6 +65,29 @@ public static void afterClass() throws Exception {
 		IResourcesSetupUtil.setAutobuild(WAS_AUTOBUILD);
 	}
 
+	/**
+	 * Test that the multi-quickfixes work as expected.
+	 * 
+	 * @param initialText
+	 *            the initial DSL text.
+	 * @param initialTextWithMarkers
+	 *            the initial DSL text enhanced by the recognized marker information.
+	 * @param resultTextWithMarkers
+	 *            the result DSL text enhanced by the recognized marker information after applying all the quickfixes.
+	 */
```

Comment:
misses `@since` annotation

Reply:
The entire [AbstractMultiQuickfixTest](https://github.com/eclipse/xtext-eclipse/blob/master/org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractMultiQuickfixTest.java#L45)class is marked with the `@since 2.23` , so there is no need to add the since tag to the method.